new/LIT7138PrLover.xml

```diff
@@ -0,0 +1,83 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7138PrLover" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Prayer of the lover</title>
+                <title xml:lang="gez" xml:id="t2">ጸሎተ፡ መስተፋቅር</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Ṣalota mastafāqər</title>
+                <title xml:lang="grc" xml:id="t3">περη αγαπης κὲ εφτήχία</title>
```

Comment:
I agree that we really need an abstract to explain that this Greek "title" comes from the manuscript itself and that the spelling has not been standardized.

The catalogue seems to have εφτηχία, with only one accent